testsuite/features/build_validation/smoke_tests/smoke_tests.template

```diff
@@ -174,6 +176,8 @@ Feature: Smoke tests for <client>
     And I wait until event "Package Install/Upgrade scheduled by <client>" is completed
 
 @skip_for_sle_micro_ssh_minion
+@skip_for_sl_micro
+  # WORKAROUND for SL Micro because of bsc#1227451 and bsc#1229567
```

Comment:
As discussed in Slack, I would personally show a red in this test case on the test report, unless this affects other test cases.
Otherwise, the test report is not reflecting the current product state.